data_loader/setup_data/collection.json

```diff
@@ -0,0 +1,93 @@
+{
+  "id":"sentinel-s2-l2a-cogs",
+  "stac_version":"1.0.0-beta.2",
```

Comment:
```suggestion
  "stac_version":"1.0.0",
```

Reply:
The sentinel-s2-l2a-cogs stac data online needs to be updated as it's still 1.0.0-beta.2

Reply:
I will fix it for our sample data